vulkan/vulkan_application.cc

```diff
@@ -96,10 +127,19 @@ VulkanApplication::VulkanApplication(
 
   if (VK_CALL_LOG_ERROR(vk_.CreateInstance(&create_info, nullptr, &instance)) !=
       VK_SUCCESS) {
+#if OS_FUCHSIA
```

Comment:
FYI: This library is only used on Fuchsia at the moment. I don't think there is a need to have Fuchsia specific guards.

Reply:
I didn't realize that. I've combined the log into the existing one and removed the #if.